components/src/main/java/com/mercadopago/sdk/android/components/Label.kt

```diff
@@ -0,0 +1,437 @@
+import com.mercadopago.sdk.android.foundation.theme.MercadoPagoTheme
+
+private const val TEXT_GROUP = "Text"
+
+/**
+ * Label Type enum class, used to determine the label type
+ * This its used to change the token color
+ */
+enum class LabelColorType {
+    /**
+     *  Primary: Label of the Primary type
+     */
+    Primary,
+
+    /**
+     *  Secondary: Label of the Secondary type
+     */
+    Secondary,
+
+    /**
+     *  Accent: Label of the Accent type
+     */
+    Accent,
+
+    /**
+     *  Negative: Label of the Negative type
+     */
+    Negative,
+
+    /**
+     *  Inverted: Label of the Inverted type
+     */
+    Inverted
+}
+
+/**
+ * Label Text type enum class, used to determine the label type
+ * This its used to change the text style
+ */
+enum class LabelTextType {
+    /**
+     * Tittle: Label of the Tittle type
+     */
+    Tittle,
+
+    /**
+     * BodyMediumSemiBold: Label of the BodyMediumSemiBold type
+     */
+    BodyMediumSemiBold,
+
+    /**
+     * BodyMediumRegular: Label of the BodyMediumRegular type
+     */
+    BodyMediumRegular,
+
+    /**
+     * BodySmallSemiBold: Label of the BodySmallSemiBold type
+     */
+    BodySmallSemiBold,
+
+    /**
+     * BodySmallRegular: Label of the BodySmallRegular type
+     */
+    BodySmallRegular,
+
+    /**
+     * BodyExtraSmallSemiBold: Label of the BodyExtraSmallSemiBold type
+     */
+    BodyExtraSmallSemiBold,
+}
+
+/**
+ * Label component- This handle the text of the components
+ * This component is used to build others components
+ * handling the text implementation
+ *
+ * @param modifier: label modifier
+ * @param text: label text
+ * @param labelTextStyle: text style, must be using
+ * @param labelColorType: type of label color
+ * @param enabled: Boolean indicates if the component its enabled
+ */
+@Composable
+fun Label(
+    modifier: Modifier = Modifier,
+    text: String = "",
+    labelTextStyle: LabelTextType = LabelTextType.Tittle,
+    labelColorType: LabelColorType = LabelColorType.Primary,
+    enabled: Boolean = true,
+) {
+    LabelComposable(
+        modifier = modifier,
+        text = text,
+        labelStyle = labelTextStyle,
+        labelColorType = labelColorType,
+        enabled = enabled,
+    )
+}
+
+@Composable
+internal fun LabelComposable(
+    modifier: Modifier,
+    text: String,
+    labelStyle: LabelTextType,
+    labelColorType: LabelColorType,
+    enabled: Boolean,
+) {
+    val color = when (labelColorType) {
+        LabelColorType.Primary -> {
+            MercadoPagoTheme.color.text.primary
+        }
+
+        LabelColorType.Secondary -> {
+            MercadoPagoTheme.color.text.secondary
+        }
+
+        LabelColorType.Accent -> {
+            MercadoPagoTheme.color.text.accent
+        }
+
+        LabelColorType.Negative -> {
+            MercadoPagoTheme.color.text.negative
+        }
+
+        LabelColorType.Inverted -> {
+            MercadoPagoTheme.color.text.inverted
+        }
+    }
```

Comment:
I agree that it would be good to enforce a text style for this component. however, the color of the text can vary a lot, so it might be more scalable to use the Color class from compose. this approach would allow us to easily swap colors when there is a new design requirement. what are your thoughts on it?